.github/dependabot.yml

```diff
@@ -80,6 +70,9 @@ updates:
         applies-to: version-updates
         patterns:
           - "*"
+        exclude-patterns:
```

Comment:
Yes good idea, however do we have any cron or something to still remind us to update deps? Otherwise we'll most likely forget

Reply:
Yeah, I'm a bit unsure of what to do here.

Dependabot will still notify us of security issues - but aside from that, I think we should probably hold updates until we really need functionality in new versions. Updating it has been a pain in the past :cry: 

Let's see how this new config works, and we can see if we want automated upgrades on a cron job later.